lib/mongoid_colored_logger/logger_decorator.rb

```diff
@@ -20,12 +20,12 @@ def initialize(logger)
 
     colorize_method = Mongoid::VERSION.to_f >= 3.0 ? :colorize_message : :colorize_legacy_message
 
-    %w[debug info warn error fatal unknown].each.with_index do |method, severity|
-      define_method(method) do |message = nil, progname = nil, &block|
+    %w[debug info warn error fatal unknown].each_with_index do |method, severity|
+      define_method(method) do |message = nil, &block|
```

Comment:
`Logger#debug` [receives](https://github.com/ruby/ruby/blob/trunk/lib/logger.rb#L490) 1 option argument and block.
And [sends](https://github.com/ruby/ruby/blob/trunk/lib/logger.rb#L491) it like in the third argument to `Logger#add`

It is not critical by no means but I think it would be better to keep code closely to original.